tests/run-tests.py

```diff
@@ -896,22 +877,93 @@ def run_one_test(test_file):
             skipped_tests.append(test_name)
             return
 
-        testcase_count.add(len(output_expected.splitlines()))
+        # Look at the output of the test to see if unittest was used.
+        uses_unittest = False
+        output_mupy_lines = output_mupy.splitlines()
+        if any(
+            line == b"ImportError: no module named 'unittest'" for line in output_mupy_lines[:3]
+        ):
+            print("error: test {} requires unittest".format(test_file))
+            print("(eg run `mpremote mip install unittest` to install it)")
+            sys.exit(1)
+        elif (
+            len(output_mupy_lines) > 4
+            and output_mupy_lines[-4] == b"-" * 70
+            and output_mupy_lines[-2] == b""
```

Comment:
Is this even needed, i.e. won't the `b"-" * 70` be sufficient?

Reply:
It's not needed, but I wanted to be strict here with the pattern matching, in case a non-unittest test outputs similar strings.